setup.py

```diff
@@ -24,49 +24,49 @@
 ]
 
 install_requires = [
-    'boto3==1.24.37',
```

Comment:
how did you figure out all these upgrades?

Reply:
From Composer setup.py.